src/jennifer/validations/macros.cr

```diff
@@ -91,6 +91,30 @@ module Jennifer
         end
       end
 
+      macro validates_composite_uniqueness(*fields, if if_value = nil)
```

Comment:
So in that case `validates_uniqueness :id, scope = [:first_name, :last_name]` would be equal to `validates_composite_uniqueness :id, :first_name, :last_name`, right?
I will gladly adapt the original *Uniqueness* validator to support this function, but since the argument type of `scope` is checked anyway, wouldn't it be more intuitive, if you could simply say `validates_uniqueness :id, :first_name, :last_name`? Otherwise the `scope` vs `field` parameters suggest another kind of precedence that is already given by the ordering of the properties. And in the macro those arguments are joined anyway. Or am I missing something right there?